07-routing-gossip.md

```diff
@@ -47,52 +47,59 @@ The willingness of the initiating node to announce the channel is signaled durin
 
 ### Requirements
 
-The `announcement_signatures` message is created by constructing a `channel_announcement` message, corresponding to the newly established channel, and signing it with the secrets matching an endpoint's `node_id` and `bitcoin_key`. The message is then sent using an `announcement_signatures`.
+The `announcement_signatures` message is created by constructing a `channel_announcement` message, corresponding to the newly established channel, and signing it with the secrets matching an endpoint's `node_id` and `bitcoin_key`. The message is then sent using an `announcement_signatures` message.
```

Comment:
This is a little awkward with the multiple uses of message, especially in the updated final sentence. Maybe something like. "This `channel_announcement` message is then sent ..." or if it's accurate perhaps "This packet is then sent ..."

Reply:
fixed